components/typography/index.css

```diff
@@ -13,6 +13,267 @@ governing permissions and limitations under the License.
+  font-family: var(--spectrum-code-s-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeXS, code-XS, true {
+  font-family: var(--spectrum-code-xs-text-font-family);
+}
+
+
+.spectrum-Typography {
+  @mixin typographyTShirtMargins .spectrum-HeadingXXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingM {}
+  @mixin typographyTShirtMargins .spectrum-HeadingS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXS {}
+
+  @mixin typographyTShirtMargins .spectrum-BodyXXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyL {}
+  @mixin typographyTShirtMargins .spectrum-BodyM {}
+  @mixin typographyTShirtMargins .spectrum-BodyS {}
+  @mixin typographyTShirtMargins .spectrum-BodyXS {}
+}
+
+
+/* T-Shirt sizes for .spectrum-Article */
+.spectrum-Article {
+  font-family: var(--spectrum-body-xl-text-font-family);
+  @mixin typographyTShirtSizes .spectrum-HeadingXXXL, heading-serif-XXXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXXL, heading-serif-XXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXL, heading-serif-XL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingL, heading-serif-L {}
+  @mixin typographyTShirtSizes .spectrum-HeadingM, heading-serif-M {}
+  @mixin typographyTShirtSizes .spectrum-HeadingS, heading-serif-S {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXS, heading-serif-XS {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXXS, heading-serif-XXS {}
+
+  @mixin typographyTShirtSizes .spectrum-BodyXXXL, body-serif-XXXL {}
+  @mixin typographyTShirtSizes .spectrum-BodyXXL, body-serif-XXL {}
+  @mixin typographyTShirtSizes .spectrum-BodyXL, body-serif-XL {}
+  @mixin typographyTShirtSizes .spectrum-BodyL, body-serif-L {}
+  @mixin typographyTShirtSizes .spectrum-BodyM, body-serif-M {}
+  @mixin typographyTShirtSizes .spectrum-BodyS, body-serif-S {}
+  @mixin typographyTShirtSizes .spectrum-BodyXS, body-serif-XS {}
+  @mixin typographyTShirtSizes .spectrum-BodyXXS, body-serif-XS {}
+
+  @mixin typographyTShirtSizes .spectrum-HeadingXXXL--light, heading-light-XXXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXXL--light, heading-light-XXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXL--light, heading-light-XL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingL--light, heading-light-L {}
+
+  @mixin typographyTShirtSizes .spectrum-HeadingXXXL--heavy, heading-heavy-XXXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXXL--heavy, heading-heavy-XXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXL--heavy, heading-heavy-XL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingL--heavy, heading-heavy-L {}
+
+  /* These don't exist */
```

Comment:
is there an issue already filed to un-comment these when DNA is released with these options?

Reply:
logged here as a subtask of the typography updates: 

[SDS-4454 Typography heading sizes: missing DNA tokens ](https://jira.corp.adobe.com/browse/SDS-4454)